src/orders/order-create-request-builder.ts

```diff
@@ -37,6 +37,16 @@ export class OrderCreateRequestBuilder {
         return this;
     }
 
+    public withAccessoriesIds(
```

Comment:
I think it would be better if this function received a list of accessories (`IAccessory`) instead of just the IDs, and that it was named `withAccessories`. It makes the code base more flexible, with a minimal cost of passing full models instead of just IDs.

Reply:
Good point. Commit 23e8722 takes this feedback into account.